apis/nucleus/src/components/selections/SelectedFields.jsx

```diff
@@ -37,7 +37,9 @@ function getItems(layout) {
   if (layout.alternateStates) {
     layout.alternateStates.forEach((s) => collect(s.qSelectionObject, fields, s.stateName));
   }
-  return Object.keys(fields).map((key) => fields[key]);
+  return Object.keys(fields)
+    .map((key) => fields[key])
+    .filter((f) => !f.selections.some((s) => s.qIsHidden));
```

Comment:
maybe just check qIsHidden on the first value in selection
(I think all of them always should have the same qIsHidden value)
```suggestion
    .filter((f) => !f.selections[0].qIsHidden);
```

Reply:
Using some here actually handles the case of an empty selections array as well, which I think can happen.